crates/symbolicator-proguard/src/interface.rs

```diff
@@ -128,6 +128,8 @@ pub struct JvmException {
 /// A JVM stacktrace.
 #[derive(Debug, Default, Clone, Deserialize, Serialize, PartialEq, Eq)]
 pub struct JvmStacktrace {
+    /// The exception that caused the stacktrace.
+    pub exception: Option<JvmException>,
```

Comment:
unsure if this is a breaking change in the symbolicator-service API (but probably not because it's an Option)?